src/DetailPage.js

```diff
@@ -1,11 +1,44 @@
-import { useEffect, useState } from 'react';
-import { useRouteMatch } from 'react-router-dom';
+import { useEffect, useState, } from 'react';
+import { useRouteMatch, useParams, useHistory } from 'react-router-dom';
+import { updateGame } from './services/fetch-utils';
 import { getGameById } from './services/fetch-utils';
 
 export default function DetailPage() {
   const [game, setGame] = useState({});
+  const match = useRouteMatch();
+  const { push } = useHistory();
+  const [title, setTitle] = useState('');
+  const [genre, setGenre] = useState('');
+  const [designer, setDesigner] = useState('');
+  const [description, setDescription] = useState('');
+  const [minPlayers, setMinPlayers] = useState(0);
+  const [maxPlayers, setMaxPlayers] = useState(0);
```

Comment:
I think instead of initializing all this state we can just use the game state that we get on load and then updating that on each input. An added benefit of that is our form can load in with the values so we can see what we want to adjust immediately